rocksdb_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then 
+        sudo apt-get install libgflags-dev
+        sudo apt-get install libsnappy-dev
+        sudo apt-get install zlib1g-dev
+        sudo apt-get install libbz2-dev
+        sudo apt-get install liblz4-dev
+        sudo apt-get install libzstd-dev
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+        brew install rocksdb
+fi
```

Comment:
Let's add an else error case saying that we don't currently support the operating system